tutorials/lsrro.ipynb

```diff
@@ -0,0 +1,5428 @@
+      " Membrane replacement factor [fraction of membrane replaced/year] \n",
+      " 0.15 \n",
+      "\n",
+      "\n",
+      "fs.ROUnits[2].area \n",
+      " Total Membrane area \n",
+      " 50.0 \n",
+      "\n",
+      "\n",
+      "fs.ROUnits[2].width \n",
+      " Membrane width \n",
+      " 5.0 \n",
+      "\n",
+      "\n",
+      "fs.ROUnits[2].mixed_permeate[0.0].pressure \n",
+      " State pressure \n",
+      " 101325 \n",
+      "\n",
+      "\n",
+      "fs.ROUnits[2].A_comp[0.0,H2O] \n",
+      " 4.2e-12 \n",
+      "\n",
+      "\n",
+      "fs.ROUnits[2].B_comp[0.0,NaCl] \n",
+      " 3.5000000000000004e-06 \n",
+      "\n",
+      "\n",
+      "fs.ROUnits[2].feed_side.spacer_porosity \n",
+      " membrane-channel spacer porosity \n",
+      " 0.85 \n",
+      "\n",
+      "\n",
+      "fs.ROUnits[2].feed_side.channel_height \n",
+      " membrane-channel height \n",
+      " 0.001 \n",
+      "\n",
+      "\n",
+      "fs.costing.reverse_osmosis.high_pressure_membrane_cost \n",
+      " Membrane cost \n",
+      " 50 \n",
+      "\n",
+      "\n",
+      "fs.ROUnits[3].area \n",
+      " Total Membrane area \n",
+      " 33.333333333333336 \n",
+      "\n",
+      "\n",
+      "fs.ROUnits[3].width \n",
+      " Membrane width \n",
+      " 5.0 \n",
+      "\n",
+      "\n",
+      "fs.ROUnits[3].mixed_permeate[0.0].pressure \n",
+      " State pressure \n",
+      " 101325 \n",
+      "\n",
+      "\n",
+      "fs.ROUnits[3].A_comp[0.0,H2O] \n",
+      " 4.2e-12 \n",
+      "\n",
+      "\n",
+      "fs.ROUnits[3].B_comp[0.0,NaCl] \n",
+      " 3.5000000000000004e-06 \n",
+      "\n",
+      "\n",
+      "fs.ROUnits[3].feed_side.spacer_porosity \n",
+      " membrane-channel spacer porosity \n",
+      " 0.85 \n",
+      "\n",
+      "\n",
+      "fs.ROUnits[3].feed_side.channel_height \n",
+      " membrane-channel height \n",
+      " 0.001 \n",
+      "\n",
+      "\n",
+      "fs.EnergyRecoveryDevices[1].control_volume.properties_out[0.0].pressure \n",
+      " State pressure \n",
+      " 7000000.0 \n",
+      "\n",
+      "\n",
+      "fs.EnergyRecoveryDevices[1].efficiency_pump[0.0] \n",
+      " 0.8 \n",
+      "\n",
+      "\n",
+      "fs.costing.energy_recovery_device.pressure_exchanger_cost \n",
+      " Pressure exchanger cost \n",
+      " 535 \n",
+      "\n",
+      "\n",
+      "fs.EnergyRecoveryDevices[3].control_volume.properties_out[0.0].pressure \n",
+      " State pressure \n",
+      " 101325 \n",
+      "\n",
+      "\n",
+      "fs.EnergyRecoveryDevices[3].efficiency_pump[0.0] \n",
+      " 0.8 \n",
+      "\n",
+      "\n",
+      "fs.feed.properties[0.0].pressure \n",
+      " State pressure \n",
+      " 101325 \n",
+      "\n",
+      "\n",
+      "fs.feed.properties[0.0].temperature \n",
+      " State temperature \n",
+      " 293.15 \n",
+      "\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "lsrro.display_operating_conditions(m)"
```

Comment:
Do we need this? At the very least, I think the name is a bit misleading since it suggests that all of these variables need to be fixed in set_operating_conditions, which is not the case. In general, it's also just not the most aesthetically pleasing thing to look at.